connect/runtime/src/test/java/org/apache/kafka/connect/runtime/SourceConnectorConfigTest.java

```diff
@@ -64,6 +67,18 @@
         return props;
     }
 
+    @Test
+    public void shouldNotFailWithExplicitlySpecifiedDefaultTopicCreationGroup() {
+        Map<String, String> props = defaultConnectorProps();
+        //Ensure default group is omitted even if its specified multiple times.
+        props.put(TOPIC_CREATION_GROUPS_CONFIG, String.join(",", DEFAULT_TOPIC_CREATION_GROUP,
+            TOPIC_CREATION_GROUP_1, DEFAULT_TOPIC_CREATION_GROUP, TOPIC_CREATION_GROUP_2));
+        props.put(TOPIC_CREATION_GROUPS_CONFIG, DEFAULT_TOPIC_CREATION_GROUP);
```

Comment:
Was this left in accidentally? Looks like it overrides the line above.

Reply:
> Was this left in accidentally? Looks like it overrides the line above, which seems more suitable for this test case.

Thanks for catching it! fixed